lib/ui/platform_dispatcher.dart

```diff
@@ -308,6 +308,15 @@ class PlatformDispatcher {
     _invoke(onMetricsChanged, _onMetricsChangedZone);
   }
 
+  // [FlutterView]s for which [FlutterView.render] has already been called
+  // during the current [onBeginFrame]/[onDrawFrame] callback sequence.
+  //
+  // The field is null outside the scope of those callbacks indicating that
+  // calls to [FlutterView.render] must be ignored. Furthermore, if a given
+  // [FlutterView] is already present in this set when its [FlutterView.render]
+  // is called again, that call must be ignored as a duplicate.
+  Set<FlutterView>? _renderedViews;
+  Set<FlutterView>? _renderedViewsBetweenCallbacks;
```

Comment:
```suggestion
  // [FlutterView]s for which [FlutterView.render] has already been called
  // during the last `_beginFrame` call.
  //
  // The field holds the result of `_renderedViews` temporarily between `_beginFrame` and `_drawFrame`, 
  // so that `_renderedViews` can be null during the gap, but still gets the previous value 
  // at the at the beginning of `_drawFrame`.
  Set<FlutterView>? _renderedViewsInPastCalls;
```